src/components/Editable.tsx

```diff
@@ -346,6 +346,20 @@ const Editable = ({
         ),
       )
 
+      if (newValue.toLowerCase().indexOf(' new thought') > 0) {
+        const [prev, next] = newValue.split('new thought')
```

Comment:
The split should also be case insensitive.

Reply:
got it